admin/customers.php

```diff
@@ -628,164 +434,96 @@ function check_form() {
+              <?php echo zen_draw_label(ENTRY_STATE, 'entry_state', 'class="col-sm-3 control-label"'); ?>
               <div class="col-sm-9 col-md-6">
-                  <?php
-                  $entry_state = zen_get_zone_name($cInfo->country_id, $cInfo->zone_id, $cInfo->state);
-                  if ($error == true) {
-                    if ($entry_state_error == true) {
-                      if ($entry_state_has_zones == true) {
-                        $zones_array = array();
-                        $zones_values = $db->Execute("SELECT zone_name
-                                                    FROM " . TABLE_ZONES . "
-                                                    WHERE zone_country_id = " . (int)zen_db_input($cInfo->country_id) . "
-                                                    ORDER BY zone_name");
-
-                        while (!$zones_values->EOF) {
-                          $zones_array[] = array('id' => $zones_values->fields['zone_name'], 'text' => $zones_values->fields['zone_name']);
-                          $zones_values->MoveNext();
-                        }
-                        echo zen_draw_pull_down_menu('entry_state', $zones_array, '', 'class="form-control"') . '&nbsp;' . ENTRY_STATE_ERROR;
-                      } else {
-                        echo zen_draw_input_field('entry_state', htmlspecialchars(zen_get_zone_name($cInfo->country_id, $cInfo->zone_id, $cInfo->state), ENT_COMPAT, CHARSET, TRUE), 'class="form-control"') . '&nbsp;' . ENTRY_STATE_ERROR;
-                      }
-                    } else {
-                      echo $entry_state . zen_draw_hidden_field('entry_zone_id') . zen_draw_hidden_field('entry_state');
-                    }
-                  } else {
-                    echo zen_draw_input_field('entry_state', htmlspecialchars(zen_get_zone_name($cInfo->country_id, $cInfo->zone_id, $cInfo->state), ENT_COMPAT, CHARSET, TRUE), 'class="form-control"');
-                  }
-                  ?>
-              </div>
-            </div>
-            <?php
-          }
-          ?>
-          <div class="form-group">
-              <?php echo zen_draw_label(ENTRY_COUNTRY, 'entry_country_id', 'class="col-sm-3 control-label"'); ?>
-            <div class="col-sm-9 col-md-6">
                 <?php
-                if ($error == true) {
-                  if ($entry_country_error == true) {
-                    echo zen_draw_pull_down_menu('entry_country_id', zen_get_countries_for_admin_pulldown(), $cInfo->country_id, 'class="form-control"') . '&nbsp;' . ENTRY_COUNTRY_ERROR;
-                  } else {
-                    echo zen_get_country_name($cInfo->country_id, false) . zen_draw_hidden_field('entry_country_id');
+                $entry_state = zen_get_zone_name($cInfo->country_id, $cInfo->zone_id, $cInfo->state);
+                if ($entry_state_has_zones == true) {
```

Comment:
@Zen4All unfortunately after this merge this line 477 now triggers the following notice which didn't occur before:
`PHP Notice: Undefined variable: entry_state_has_zones in /admin/customers.php on line 477.`